src/ocean/sys/HomeFolder.d

```diff
@@ -187,7 +187,7 @@ version (Posix)
 
             // Obtain info from database.
             passwd *verify;
-            core.stdc.errno.errno(0);
+            core.stdc.errno.errno = 0;
```

Comment:
Explanation would be nice

Reply:
`sys/HomeFolder`. OMG, Tango had a whole module to get the user's home "folder" ("folder" is SOOO windows :roll_eyes: ).

Anyway...